src/_gettsim/config.py

```diff
@@ -179,5 +181,8 @@ def set_array_backend(backend: str):
     "kind_unterh_anspr_m": float,
     "kind_unterh_erhalt_m": float,
     "steuerklasse": int,
+    "erwerbsgemindert": bool,
+    "erwerbsm_rente_mit63": bool,
```

Comment:
```suggestion
    "erwerbsm_rente_mit_63": bool,
```
Won't flag again and this should change, anyhow. But this is what was used above in the list of input variables.